src/config.rs

```diff
@@ -1330,11 +1330,17 @@ impl TiKvConfig {
         }
 
         if last_cfg.storage.data_dir != self.storage.data_dir {
-            return Err(format!(
-                "storage data dir have been changed, former data dir is {}, \
-                 current data dir is {}, please check if it is expected.",
-                last_cfg.storage.data_dir, self.storage.data_dir
-            ));
+            // In v3 the default value of storage.data-dir changed
```

Comment:
what does V3 mean here?

Reply:
Configuration version 3 (Do we document how we support old/new configs?)

Reply:
I think we don't have

Reply:
I meant TiKV version 3, the next release. I will clarify in the code.